articles/libraries/lock-android/index.md

```diff
@@ -1,478 +1,248 @@
 
-* **Integrates** your Android app with **Auth0**.
-* Provides a **beautiful native UI** to log your users in.
-* Provides support for **Social Providers** (Facebook, Twitter, etc.), **Enterprise Providers** (AD, LDAP, etc.) and **Username & Password**.
-* Passwordless authentication using **SMS or Email**.
-
-## Additional Documents
-
-<ul>
-<% _.forEach(_.sortBy(articles.findByHash('libraries/lock-android').items, 'toc_title'), function(article) { %>
-  <% if (article.toc_title) { %>
-  <li>
-    <span><a href="<%- '/docs' + article.url %>"><%- article.toc_title %></a>
-    <% if (article.description) { %>
-      - <%- article.description %>
-    <% } %>
-    </span>
-  </li>
-  <% } %>
-<% }); %>
-</ul>
+Get started using Lock for Android below, or if you're looking for a specific document, try the listing of [additional documents](#additional-documents) related to Lock for Android!!
 
 ## Requirements
 
 Android API level 15+ is required in order to use Lock's UI.
-If you'll create your own API and just call Auth0 API via the `com.auth0.android:core:1.13.+`, the minimum required API level is 9.
+If you intend to create your own API and just call Auth0 API via the `com.auth0.android:auth0:1.1.0`, the minimum required API level is also 15+.
 
-## Install
+## Installation
 
 Lock is available both in [Maven Central](http://search.maven.org) and [JCenter](https://bintray.com/bintray/jcenter). To start using *Lock* add these lines to your `build.gradle` dependencies file:
 
 ```gradle
-compile 'com.auth0.android:lock:1.13.+'
-```
-
-Once it's installed, you'll need to configure LockActivity in your`AndroidManifest.xml`, inside the `application` tag:
-
-```xml
-<!--Auth0 Lock-->
-<activity
-  android:name="com.auth0.lock.LockActivity"
-  android:theme="@style/Lock.Theme"
-  android:screenOrientation="portrait"
-  android:launchMode="singleTask">
-  <intent-filter>
-    <action android:name="android.intent.action.VIEW"/>
-    <category android:name="android.intent.category.DEFAULT"/>
-    <category android:name="android.intent.category.BROWSABLE"/>
-    <data android:scheme="a0INSERT_YOUR_APP_CLIENT_ID" android:host="YOUR_ACCOUNT_NAME.auth0.com"/>
-  </intent-filter>
-</activity>
-<meta-data android:name="com.auth0.lock.client-id" android:value="@string/auth0_client_id"/>
-<meta-data android:name="com.auth0.lock.domain-url" android:value="@string/auth0_domain"/>
-<!--Auth0 Lock End-->
+compile 'com.auth0.android:lock:2.0.0'
 ```
 
-> The value `@string/auth0_client_id` is your application's clientID and `@string/auth0_domain` is your tenant's domain in Auth0, both values can be found in your app's settings.
-> The final value of `android:scheme` must be in lowercase
+_You can check for the latest version on the repository [Releases](https://github.com/auth0/Lock.Android/releases) tab or in [Maven](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22lock%22%20g%3A%22com.auth0.android%22)_
```

Comment:
Let's add JCenter too, in Android is more common to use it since it's the default repository for gradle